content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,243 @@
+============
+Event booths
+============
+
+The Odoo *Events* application provides users with the ability to create event booths, sell their
+availability, and manage their reservations.
+
+Configuration
+=============
+
+In order to create, sell, and manage booths for events, the *Booth Management* feature must be
+activated.
+
+To do that, navigate to :menuselection:`Events app --> Configuration --> Settings`, and tick the
+checkbox beside :guilabel:`Booth Management`. Then, click :guilabel:`Save`.
+
+.. image:: event_booths/booth-management-setting.png
+   :align: center
+   :alt: The Booth Management setting in the Odoo Events application.
+
+.. important::
+   When the :guilabel:`Booth Management` setting is activated, a new *Product Type* becomes
+   available on all product forms: *Event Booth*.
+
+   This is important because every created booth **must** be assigned a *Booth Category* on their
+   respective booth form, and every booth category **must** have an *Event Booth* product assigned
+   to it.
+
+Booth categories
+================
+
+With the *Booth Management* setting activated in the *Events* app, a *Booth Categories* option
+becomes available in the :guilabel:`Configuration` menu.
+
+To access the :guilabel:`Booth Category` dashboard, go to :menuselection:`Events app -->
+Configuration --> Booth Categories`. Doing so reveals a list of all the booth categories that have
+been created in the database.
```

Comment:
Totally optional: slight rewording for brevity

With the Booth Management setting activated in the Events app, the Booth Categories option appears in the :guilabel:Configuration menu.

To access it, go to :menuselection:Events app --> Configuration --> Booth Categories, revealing a list of all created booth categories.